### ISSUE TITLE
Readme incomplete, features missing

### ISSUE THREAD

**Issue by sp00n** [mann1x/CPUDoc#12]
The readme is missing what it actually does, it does mention the "following" features, but they are never listed.

I assume it should list those from the [overclock.net](https://www.overclock.net/threads/cpudoc-little-cpu-helper-tool-with-some-exclusive-features.1802081/) thread:
> CPUDoc is a little utility that comes with some exclusive features:
> - SysSetHack - Use the CpuSet system bitmask manipulation to get a smoother and faster experience similar to SMT/HyperThreading off, while still keeping the benefit on being available.
> - PowerActive - Dynamic power plan manipulation, comes with 2 ultra low power consumption standby mode, Light and Deep Sleep.
> - NumaZero - Only use a subset of the cores, either set by Auto or manually the first 2/4/6/8. Auto should select same CCD/CCX or P-Cores.

Or whatever else it does as well, I was confused since nothing was mentioned in the readme.

**Comment by sp00n** [comment]
I also just noticed that the link to the AMD thread further down below links to the one for Intel and vice versa.

**Comment by mann1x** [comment]
> Or whatever else it does as well, I was confused since nothing was mentioned in the readme.

You are right that was meant to be there, maybe I never put it there or removed it by mistake.

> I also just noticed that the link to the AMD thread further down links to the one for Intel and vice versa.

Yes, had noticed this already but didn't commit yet.
Hopefully I'll restarting working on it after this summer but I'll try to push the updates asap.

Thanks for taking the time to let me know!
